assets/chezmoi.io/docs/user-guide/tools/merge.md

```diff
@@ -67,7 +67,7 @@ To use [VSCode][vscode] as the merge tool, add the following to your config:
     command = "bash"
     args = [
         "-c",
-        "cp {{ .Target }} {{ .Target }}.base && code --new-window --wait --merge {{ .Destination }} {{ .Target }} {{ .Target }}.base {{ .Source }}",
+        "cp \"{{ .Target }}\" \"{{ .Target }}.base\" && code --new-window --wait --merge \"{{ .Destination }}\" \"{{ .Target }}\" \"{{ .Target }}.base\" \"{{ .Source }}\"",
```

Comment:
Thanks, for this. However it's better to use the `quote` template function, e.g. `{{ .Target | quote }}`, than adding quotes manually at each end.